po/po4a.conf

```diff
@@ -0,0 +1,12 @@
+[po4a_langs] es
```

Comment:
We should make this REUSE compliant as well. Not a big deal on such files, can do it later